src/nvim/ui.c

```diff
@@ -220,11 +221,13 @@ void ui_busy_stop(void)
 void ui_attach_impl(UI *ui)
 {
   if (ui_count == MAX_UI_COUNT) {
+    apply_autocmds(EVENT_UIATTACHFAILED, NULL, NULL, FALSE, curbuf);
```

Comment:
If this ever happens we will just raise the max. I don't think we need an event dedicated to it.